### ISSUE TITLE
Update obsoleted target annotation query from playground

### ISSUE THREAD

**Issue by d0choa** [opentargets/issues#3043]
API has changed and the target annotation query does not work.

target > tractability > id

is now

target > tractability > label

<img width="1498" alt="Screenshot 2023-08-21 at 11 23 35" src="https://github.com/opentargets/issues/assets/5097586/16f7d672-10e9-4835-82ab-67d7df5147b8">

**Comment by buniello** [comment]
<details>
  <summary>Correct API query</summary>

```
query targetAnnotation {
  target(ensemblId: "ENSG00000169083") {
    id
    approvedSymbol
    tractability {
      modality
      label
      value
    }
    safetyLiabilities {
      event
      eventId
      biosamples {
        cellFormat
        cellLabel
        tissueLabel
        tissueId
      }
      effects {
        dosing
        direction
      }
      studies {
        name
        type
        description
      }
      datasource
      literature
    }
  }
}
```
</details>

@chinmehta could you please replace the existing `target annotation` example query with the correct one above

**Comment by fedde-s** [comment]
I just found this, checked whether it was already being tracked here and found that it was… since yesterday :joy:

**Comment by d0choa** [comment]
@opentargets/fe-team testing would really help with these kinds of things

**Comment by carcruz** [comment]
I can confirm that it is merged on the latest `main`

**Comment by prashantuniyal02** [comment]
Hi @fedde-s , just wanted to inform you that this has been resolved in the latest Platform 23.09 release. Hence closing the issue.

**Comment by fedde-s** [comment]
Sorry for my delayed reply—I can also confirm that it's fixed, even when running the code outside of EBI's infrastructure :smile:

Nice and tidy!
